python/cucim/src/cucim/clara/converter/tiff.py

```diff
@@ -73,11 +73,12 @@ def svs2tif(
     logger.info("      num_workers: %d", num_workers)
     logger.info("      compression: %s", compression)
     logger.info("  output filename: %s", output_filename)
-
+    compressionargs = None
```

Comment:
Do we want this to be user facing? Or do we want to keep this internal for now?

Reply:
If we make it user facing, we may need to add some additional validation on the input. I would be fine with just keeping it internal for now.

@gigony  do you have a preference?

Reply:
@grlee77 @jakirkham 
It seems appropriate to update it now since this API change has been in effect since July 2022.
https://github.com/cgohlke/tifffile/blame/ec6be6289db1b8b7327bb98816a764c95b9b7299/README.rst#L519

Reply:
@grlee77 were there particular thoughts you had on what input validation would look like?

Wonder if tifffile's own validation would work for us